### PR TITLE
FIRST PR --- splash and mainActivity

### PR DESCRIPTION
**Pull request by AndresConfortiGL**
## Description
1th PR - Added a splash screen, a main activity, and applied unit testing.

## Story Type
- New feature :white_check_mark:
- Change request
- Bug fix
- Tech

## How Can This Be Tested?
By running the app, you will be displayed an splash screen followed by the main activity.
Unit testing can be seen by running the coverage tests.

## Evidences
| Pixel 3a | Nexus S |
| ------------- | ------------- |
|pixel3|nexus|

| Coverage Test  |
| ------------- |
|coverage|

## Checklist

- All unit Test passed :white_check_mark:

## Tested on
The project was tested on Android 8.0 / API 26 using the following devices:

- Pixel 3a
- Nexus S

## Package Structure
| Structure |
| ------------- |
|packages|

### CODE REVIEW

**Comment by jnicolaslaurenti** [comment]
Remove .png files, please

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="default_text_size">20dp</dimen>
```
Comment (SartiMau) [inline app/src/main/res/values/dimens.xml:3]: this should be a sp, not a dp

**Review by SartiMau — CHANGES_REQUESTED** [review]
1. Please format the code properly
2. Both .idea files should not be here
3. Do not let png images, transformed them to a webp

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (AndresConfortiGL) [reply app/src/main/res/values/dimens.xml:3]: fixed

**Pull request closed without merging**